src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.MultiAgent.cs

```diff
@@ -1244,6 +1228,48 @@ private static ulong TransferDataFromRequestStream(IntPtr buffer, int length, Ea
                 return Interop.Http.CURL_READFUNC_PAUSE;
             }
 
+            /// <summary>
+            /// Given a completed task used to retrieve the content stream asynchronously, extracts the stream,
+            /// stores it into <see cref="EasyRequest._requestContentStream"/>, and does an initial read on it.
+            /// </summary>
+            private static Task<int> StoreRetrievedContentStreamAndReadAsync(
+                Task<Stream> readStream, EasyRequest easy, EasyRequest.SendTransferState sts, int length)
+            {
+                Debug.Assert(readStream.IsCompleted, $"Expected readStream to be completed, got {readStream.Status}");
+                try
+                {
+                    MultiAgent multi = easy._associatedMultiAgent;
+                    multi.EventSourceTrace("ReadAsStreamAsync completed: {0}", readStream.Status, easy: easy);
```

Comment:
(The tracing implicitly traces the calling function name, so I'll just remove the name from here; it'll be clear in the trace what's happened.)